### ISSUE TITLE
Indicator "sports facilities accessibility" does not work

### ISSUE THREAD

**Issue by gamalielpalomo** [CityScienceLab-GDL/cityscope-ccu#16]
![image](https://user-images.githubusercontent.com/19692636/235958907-f954d7bd-7790-4ca9-9ada-4f08a1c9ce6c.png)
Nil value is detected, this has been solved in the past with other indicators. Do the same approach to solve it.

**Comment by gamalielpalomo** [comment]
Implemented in the last version. I will close it.
